pytensor/link/jax/dispatch/tensor_basic.py

```diff
@@ -68,6 +75,57 @@ def join(axis, *tensors):
     return join
 
 
+@jax_funcify.register(Split)
+def jax_funcify_Split(op: Split, node, **kwargs):
+    _, axis, splits = node.inputs
+    try:
+        constant_axis = get_scalar_constant_value(axis)
+    except NotScalarConstantError:
+        constant_axis = None
+        warnings.warn(
+            "Split node does not have constant axis. Jax implementation will likely fail"
+        )
+
+    try:
+        constant_splits = np.array(
+            [
+                get_scalar_constant_value(splits[i])
+                for i in range(get_vector_length(splits))
+            ]
+        )
+    except (ValueError, NotScalarConstantError):
+        constant_splits = None
+        warnings.warn(
+            "Split node does not have constant split positions. Jax implementation will likely fail"
+        )
+
+    def split(x, axis, splits):
+        if constant_axis is not None:
+            axis = constant_axis
+        if constant_splits is not None:
+            splits = constant_splits
+            cumsum_splits = np.cumsum(splits[:-1])
```

Comment:
Why use the np version here?

Reply:
Because the jax one leads to a tracedarray that can't be used in `jnp.split` anymore.

Reply:
Maybe add a comment that this is a fall-back.